test/strategies/UniversalStrategyTest.ts

```diff
@@ -74,13 +75,23 @@ async function universalStrategyTest(
       deployInfo.underlying = underlying;
       deployInfo.vault = vault;
       deployInfo.strategy = strategy;
-
+      let calculator =  deployInfo?.tools?.calculator as PriceCalculator;
+      // temporary solution until new Price calculator deployed
+      const isBTP = await StrategyTestUtils.isBalancerLP(user, underlying);
+      if (isBTP && deployInfo?.core?.controller != null && deployInfo.tools?.calculator != null) {
+        const newCalculator = (await DeployerUtils.deployPriceCalculator(signer, deployInfo.core.controller.address))[0];
+        for (const bptLp of FtmAddresses.BPT_LPs) {
+          await newCalculator.registerBPT(bptLp);
```

Comment:
move registration to deploy function

Reply:
refactored this logic. Not needed anymore.